solutions/ccc-Twenty-four.mdx

```diff
@@ -7,7 +7,10 @@ author: Paul Chen
 
 ## Explanation
 
-We can complete search through all possible ways to create an arithmetic expression with a card hand. For each permutation of the card hand, we try every way of placing arithmetic operators. The arithmetic expression with the maximum value that is not greater than 24 is our answer.
+We can complete search through all possible ways to create an arithmetic expression with a card hand. 
+For each permutation of the card hand, we try every way of placing arithmetic operators. 
+Note that there are two possible ways that parentheses can be arranged: ((( ) ) ), or (( ) ( )), so we must try both cases.
```

Comment:
put the parenthesis parts in single line code?

Reply:
oh good idea